README.md

```diff
@@ -95,6 +99,29 @@ vmm-reference                           \
     --kernel path=/path/to/kernel/image
 ```
 
+## Testing
+
+The reference VMM is, first and foremost, a vehicle for end-to-end testing of
+`rust-vmm` crates. Each crate must contain individual functional and
+performance tests that exercise as wide a range of use cases as possible; the
+reference VMM is not meant to reiterate on that, but to validate all the pieces
+put together. The two Rust integration tests (one for each supported kernel
+image format, i.e. ELF and bzImage) exercise the only public function of the
+`Vmm` object, `run()`.
+The Python integration tests make use of the VMM in varied configurations that
+aren’t overly complex and illustrate realistic use cases (e.g. one test runs a
+VMM with a virtio block device, one test will run a VMM with PCI, etc.).
+
+To be able to successfully run all the tests in this repo, there are a couple of
+resources that need to be found in `/tmp`. To generate the resources at the
+expected locations, you can run the following command:
+`sudo ./../resources/make_test_resources.sh`.
```

Comment:
You can also add that you need to `cd resources` first and then run this command from there. I would also like to see how to run the tests (via `cargo test`).

Reply:
Oops, I thought I'm in `docs`, fixed! Also added more details about how to run the tests, thanks for the feedback!